### PR TITLE
last commit probably

### PR DESCRIPTION
**Pull request by hacknorris-aka-penguin**
(too lazy to add more lool)

for the now docs are made and added few calculators more ;p

### CODE REVIEW

File: package/index.js [hunk 1]
```diff
@@ -195,6 +201,28 @@ function timeCalc(seconds) {
...
+    return r + r + g + g + b + b;
+  });
+
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  return result ? {
+    r: parseInt(result[1], 16),
+    g: parseInt(result[2], 16),
+    b: parseInt(result[3], 16)
+  } : null;
+}
```
Comment (Susmita-Dey) [inline package/index.js:224]: Here, you have included rgb to hex and vice-versa. As far as I know, calculators don't have these things. These are color code converters. Would it make this calculator unique from others?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: package/index.js [hunk 2]
```diff
@@ -129,6 +129,12 @@ function toTri(num) {
   return num.toString(3);
 }
 
+// changing number to a string
+
+function toStr(num) {
+return num.toString();
+}
+
```
Comment (Susmita-Dey) [inline package/index.js:137]: What's its use in a calculator?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: package/index.js (hunk 1 above)
Comment (hacknorris-aka-penguin) [reply package/index.js:224]: yes i did it to differentiate just that colors are only number representations to them so i added. not saying that there exist color calculators (not yet implemented)...

File: package/index.js (hunk 2 above)
Comment (hacknorris-aka-penguin) [reply package/index.js:137]: isnt it also about numbers?

Comment (Susmita-Dey) [reply package/index.js:137]: Here, you're converting a number to a string. What a user will do with that in terms of calculation?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: docs/README.md [hunk 3]
```diff
@@ -1,14 +1,184 @@
...
+- GBP
+- CNY
+- JPY
+- CAD
+- AUD
+- AED
+
+<!-- same as  calcLoan. send help !!! -->
+
+
```
Comment (Susmita-Dey) [inline docs/README.md:184]: LGTM.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Comment (hacknorris-aka-penguin) [reply docs/README.md:184]: =]

File: package/index.js (hunk 2 above)
Comment (hacknorris-aka-penguin) [reply package/index.js:137]: well... okay, edit it.

**Comment by Susmita-Dey** [comment]
@hacknorris-aka-penguin Currently, there's no need for the functions you have created. So, I'm closing this issue. Will think about it in the future. Thanks :)

**Comment by hacknorris-aka-penguin** [comment]
AND DOCS?!

**Pull request closed without merging**